vendor/bat-native-ledger/src/bat/ledger/internal/request/request_util.h

```diff
@@ -18,7 +18,8 @@ enum class ServerTypes {
   kPublisher,
   kPromotion,
   kPayments,
-  kAPI
+  kAPI,
+  kPrivateCdn
```

Comment:
should this be `kPrivateCDN`? as it's short version

Reply:
done